### PR TITLE
2018

### PR DESCRIPTION
**Pull request by mrshu**

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 from eprihlaska import app
-app.run(debug=True)
+app.run(debug=True, host='0.0.0.0')
```
Comment (Adman) [inline run.py]: Do we want to commit also host setting?

Comment (mrshu) [reply run.py]: Good catch!

Can you think of a good reason why that would not be a good idea?

Comment (Adman) [reply run.py]: Not really. As I'm thinking about it, in production we do not run the app right from run.py, but rather use wsgi? ,  so we can keep the host setting there.

Comment (mrshu) [reply run.py]: That is indeed the case -- I do think we can easily keep the `host` setting here.

**Pull request merged**